flavoUr/static/flavoUr/css/style.css

```diff
@@ -0,0 +1,23 @@
+body{
+  text-align:center;
```

Comment:
ok now i'm being CRAZY anal (so sorry in advance) -- but we should keep the code styling consistent. For the CSS should we do:

```element-name {

}```

as opposed to

```element-name{

}```

What do you think? We can start making a project code style guide if we wanna be all official about it?